unfoldingWord/translationCore
# Fix For Errors When Opening Non-Compliant Projects

#### This pull request addresses:
This PR addresses a bug where if you open a project that manages to get into the projects folder, the alert dialog box works based on corresponding error it was caused from.


#### How to test this pull request:
To test this try opening an usfm file by copying it to the projects folder and selecting it. You should get an error.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3384)
